### ISSUE TITLE
pylint does not seem to respect custom isort sections

### ISSUE THREAD

**Issue by webknjaz** [PyCQA/pylint#3817]
### Steps to reproduce

I have third-parties divided into several sections in isort configuration like this:
```ini
# .isort.cfg
default_section = THIRDPARTY
known_frameworks = towncrier, sphinx
sections = STDLIB, TESTING, FRAMEWORKS, THIRDPARTY, FIRSTPARTY, LOCALFOLDER
```

And when I run pylint on a module that has
```python
from towncrier._settings import load_config_from_file


try:
    from towncrier.build import find_fragments  # noqa: WPS433
except ImportError:
    from towncrier import find_fragments  # noqa: WPS433, WPS440

from docutils import statemachine
```
it complains that docutils import should be before that first towncrier import.

It seems like pylint does not understand that there are custom sections and tries to shove everything into third-party.

Side note: does pylint understand `# isort: split`? I guess not.

### Current behavior

```console
src/sphinxcontrib/towncrier/__init__.py:29:0: C0411: third party import "from docutils import statemachine" should be placed before "from towncrier._settings import load_config_from_file" (wrong-import-order)
```

### Expected behavior

There should be no conflict with isort, hence no error

### pylint --version output
```console
pylint 2.6.0
astroid 2.4.2
Python 3.7.1 (default, Jan 28 2019, 08:25:13) 
[GCC 7.3.0]
```

**Comment by Pierre-Sassoulas** [comment]
pylint uses isort internally. If you're using isort with a proper configuration file yourself, you should probably disable ` wrong-import-order` or other import messages in pylint, because pylint is bound to be lagging behind the features of isort, and because isort is already fixing import order automatically for you. See #3722, #3725  and in particular [dbaty very detailed comment](https://github.com/PyCQA/pylint/pull/3725#issuecomment-674520772) : 

> If you need something special, you need to disable the wrong-import-order check in pylint and run isort manually with a proper isort configuration file.

There is a discussion to propose default configuration for such cases in #3517 in order for this problem to not appear in the first place.
